BE/Artin.BringAuto.MQTTClient/SendCarRouteService.cs

```diff
@@ -58,7 +57,20 @@ public async Task SendCarRoute(string company, string car)
                         if (order.ToStationStatus != Shared.Enums.OrderStopStatus.Done)
                             msg.Command.CarCommand.Stops.Add(new Stop() { To = order.To.Name });
                     }
-                    msg.Command.CarCommand.Route = await carRepository.GetCarRoute(company, car);
+                    var carTrack = await carRepository.GetCarTrack(company, car);
+                    if (carTrack is not null)
+                        msg.Command.CarCommand.Route = carTrack.Name ?? string.Empty;
+                    if (carTrack?.RouteStops is not null)
+                        msg.Command.CarCommand.RouteStations.Add(
+                            carTrack.RouteStops.Select(s => new Station()
+                            {
+                                Name = s.Name ?? string.Empty,
+                                Position = new Station.Types.Position()
+                                {
+                                    Latitude = s.Latitude,
+                                    Longitude = s.Longitude
```

Comment:
Altitude have to be also set, please use default value 0. If altitude is not set we are not able to parse command at client side.